#!/usr/bin/env python

PACKAGE = 'jsk_arc2017_common'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("target_location", str_t, 0, "target location", "")

exit(gen.generate(PACKAGE, "jsk_arc2017_common", "CandidatesPublisher"))
